client/src/App.js

```diff
@@ -5,7 +5,7 @@ import { theme } from "./themes/theme";
 import Signup from "./pages/Signup";
 import Login from './pages/Login'
 import Main from './pages/Main'
-import Header from "./components/Header"
+import Header from "./Components/Header"
```

Comment:
I think we would better use lowercase.
Sorry it is me who started it...

Reply:
done!